studio/stores/jsonSchema/auth_gotrue_config.json

```diff
@@ -396,6 +397,11 @@
       "title": "Minimum password length",
       "type": "integer"
     },
+    "SECURITY_UPDATE_PASSWORD_REQUIRE_REAUTHENTICATION": {
+      "title": "Update password requires reauthentication",
```

Comment:
Hmm just wondering if we can reword this - language is a lil lengthy for a toggle 

"Enable strict password updates" maybe?